src/database/repository.js

```diff
@@ -162,27 +162,6 @@ class Repository {
       type: QueryTypes.SELECT
     });
   }
-
-  async getPICInfo(onlyTokens) {
```

Comment:
We should keep this function in the codebase

Reply:
it's duplicated. Expand above and you'll see